apps/warp/warp.talon

```diff
@@ -0,0 +1,9 @@
+os: mac
+app: dev.warp.Warp-Stable
```

Comment:
This won't work I think, or at least isn't the accepted way of [matching a mac bundle](https://talon.wiki/unofficial_talon_docs/#context-header). You want `app.bundle: dev.warp.Warp-Stable` I think.